native-filesystem/source/native_filesystem.cc

```diff
@@ -229,30 +229,29 @@ namespace nativefs {
 	/**
 	* Read the contents of given block id.
 	**/
-	bool NativeFS::getBlock(uint64_t id, std::string& blk) {
-		// Look in map and get filename
-		block_info info;
-		{
-			std::lock_guard<std::mutex> lock(listMtx);
-			// Look up the block info for this id.
-			bool found = false;
-			for (size_t i = 0; i < BLOCK_LIST_LEN; i++) {
-				if (blocks[i].blockid == id && !blocks[i].free) {
-					info = blocks[i];
-					found = true;
-					break;
-				}
-			}
-			if (!found) {
-				return false;
+	bool NativeFS::getBlockInfo(uint64_t id, block_info& info) {
```

Comment:
the repl-inter-datanode has defined a function with the same signature as fetchBlock, so that's something to watch in the merge order between these two